spec/menu_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) {described_class.new}
+  let(:default_menu) { {ribs: 3,
+                        beef: 4,
+                        rolls: 3,
+                        chips: 2,
+                        pies: 5}
+                     }
```

Comment:
Lint/BlockAlignment: `}` at 10, 21 is not aligned with `let(:default_menu) { {ribs: 3,` at 5, 2.